website/package.json

```diff
@@ -6,6 +6,7 @@
     "docusaurus": "docusaurus",
     "start": "docusaurus start",
     "build": "docusaurus build",
+    "build:analyze": "docusaurus build --bundle-analyzer",
```

Comment:
Also added this command to analyze the bundle size with `npm run build:analyze`. If you want me to remove it let me know